### PR TITLE
Add default registration values to Registration Page when available

### LINKED ISSUE #159: [Integration] Make Registration page show previously used IPs
image

AC: Currently, the registration page does not show previously used IP addresses. Show them.

### PR DESCRIPTION
**Pull request by atharvakale343**
Closes #159 

image

### CODE REVIEW

File: src/main/database/sqlite-db.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ class SQLiteDB {
       info('Connecting to SQLite database');
       await this.connection.authenticate();
       await this.initTables();
-      await this.connection.sync();
+      await this.connection.sync({ force: true });
```
Comment (ShriramG24) [inline src/main/database/sqlite-db.ts:22]: Is `{force: true}` needed? Based on what I see online, it drops the tables before creating them each time, which means the data would always be reset on startup.

Comment (atharvakale343) [reply src/main/database/sqlite-db.ts:22]: `alter: true` is broken and doesn't work. `force: true` is not recommended for production, but creates the column I need. We need to use migrations at some point.

Comment (ShriramG24) [reply src/main/database/sqlite-db.ts:22]: Alright, fair.

Comment (atharvakale343) [reply src/main/database/sqlite-db.ts:22]: Steps to reproduce broken `alter: true`:
1. Stop any dev run process
2. Switch to main branch
3. Delete your `rbox-data.db` (if you know where its located)
4. Start a `npm run start` in the main branch
5. Stop the dev process
6. Switch to this branch
7. Change `{ force: true }` to `{ alter: true }`
8. Start a `npm run start` in the main branch
9. See errors in terminal

**Review by ShriramG24 — APPROVED** [review]
Looks good to me.

**Pull request merged**